Automattic/jetpack
# Refactor/subscriber counts into redux store

Closes 88-gh-Automattic/gold

## Proposed changes:

* Fetches subscriber counts from data store instead of passing it to the child component as props from parents.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* On an installation of WordPress, make sure the Jetpack Beta plugin is installed, and then activate this branch.
* Check this patch out to sun on sandbox.
* Ensure subscribe and public-api are pointed to sandbox.
* Go to [the earn settings page](https://wordpress.com/earn/payments/) and ensure Stripe is setup in test mode.
* Create a new page or post and add the subscribe block.
* Publish the page and try subscribing. Confirm the block works as expected.
* Return to edit the page with the subscription button. Click on the "Newsletter Visibility" drop-down and select "paid subscribers". You should not see a message indicating that you don't have any subscribers. Instead, you should see a message saying this will be sent to your _n_ subscribers.
